src/Http/Http.Results/src/TypedResults.cs

```diff
@@ -139,9 +140,7 @@ public static ContentHttpResult Text(string? content, string? contentType, Encod
     /// <param name="contentType">The content type (MIME type).</param>
     /// <param name="statusCode">The status code to return.</param>
     /// <returns>The created <see cref="Utf8ContentHttpResult"/> object for the response.</returns>
-#pragma warning disable RS0027 // Public API with optional parameter(s) should have the most parameters amongst its public overloads.
```

Comment:
Why were we able to remove these suppressions? Can we do the same in Results.cs?